pyproject.toml

```diff
@@ -21,6 +21,7 @@ classifiers = [
 ]
 
 dependencies = [
+  "coverage",
```

Comment:
It's better to avoid such dependencies as you don't want to pollute the venv in production, lambdas, ...

That being said, a comment in the toml file would have been appropriate. I should have done it.